src/components/MarkdownRenderer/MarkdownRenderer.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Markdown from 'markdown-to-jsx';
+
+interface MarkdownRendererProps {
+    content: string;
+    className?: string;
+}
+
+const MarkdownRenderer: React.FC<MarkdownRendererProps> = ({
+    content,
+    className,
```

Comment:
why are we passing style/class name as a prop?

Reply:
For Customization and Reusability.

Reply:
how does passing classname increase reusability, also where else do we plan to use this?

Reply:
When we pass the class name we can set the custom style and we can use it in other places like showing the pull requests etc

Reply:
Can you tell me the issue when I am passing the class name?

Reply:
So when we pass the class name There are too many things to concern ourselves with, it would be easier for developers if they call on the markdown tag and just pass in content and it automatically renders it. As remembering class names will be hard.

Reply:
I don't think it creates any issues because we are using these components for different purposes currently we are using for two purposes The first one is for the title and the second one is for the body of we pass the default class name so we can't customize the style of you want to remove the class name from propa then we have to create the two markdown renderer components.  What do you think ?